utils/actions/detectConsoleLogs.ts

```diff
@@ -107,7 +124,10 @@ export default async function detectConsoleLogs({
     Python comments start with #.
     Other console functions such as console.info() shouldn't be counted as console logs.
     Ignore code comments from this analysis. 
-    If there is a console log, print "true", else print "false"`;
+    If there is a console log, return "true", else return "false".
+    If you return true, return a string that that has 2 values: result (true) and the line of code.
+    The line value, is the actual line in the file that contains the console log.
+    For example: true,console.log("hello world");`;
```

Comment:
Do we _really_ need to use GPT for this?
Wouldn't a list of console logs per lang be faster and less prone to failing?

Reply:
Faster yes, less prone to failing no unless we build a regex per language or use tree-sitter. It's a trade-off. 

But also think that parsing code with GPT is just the start of building our own rules engine. Is the code clean or not? Does it have too much cyclomatic complexity? Is there some redundancy? Is it sub-optimal to run that piece of code client-side? Basically, any rule that we can think of. Taking advantage of the indexed context in the future. All within the same function. 

Therefore, it's a good idea to fit it in like this (and eventually change the filename from `detecConsoleLogs.ts `to something like `parseThroughRulesEngine.ts`).

Reply:
Ok yeah, I see the point. 
Not a fan, but off it goes ⛵